cluster/src/main/java/com/hartwig/pipeline/execution/vm/storage/LocalSsdStorageStrategy.java

```diff
@@ -16,10 +16,10 @@ public LocalSsdStorageStrategy(int numberOfSsdDevices) {
     @Override
     public List<String> initialise() {
         String raidDevice = "/dev/md0";
-        String mdadm = format("mdadm --create %s --level=0 --raid-devices=%d ", raidDevice, numberOfSsdDevices);
+        StringBuilder mdadm = new StringBuilder(format("mdadm --create %s --level=0 --raid-devices=%d ", raidDevice, numberOfSsdDevices));
```

Comment:
I _knew_ you would ask. Intellij was yelling at me! It came up in the code analysis because it's inside a loop apparently.

The discussion in the accepted answer here is relevant I think:

https://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java